### PR TITLE
Search Page for recipes

### PR DESCRIPTION
**Pull request by rudream**
Added recipes search page. If you click on a recipe, it will take you to `/chefprofile/:chefId/:recipeId`

recipespage

### CODE REVIEW

File: client/public/index.html [hunk 1]
```diff
@@ -3,11 +3,8 @@
   <head>
     <meta charset="utf-8" />
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
-    <link rel="preconnect" href="https://fonts.gstatic.com" />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap"
-      rel="stylesheet"
-    />
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;600;700&display=swap" rel="stylesheet">
```
Comment (dproc96) [inline client/public/index.html]: Can you fix the spacing here?

File: client/src/components/SearchRecipesCard.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+            ${recipePrice}
+          </Typography>
+        </CardContent>
+        <Divider />
+        <CardContent style={{ paddingBottom: "20px", paddingTop: "20px" }}>
+          <Grid container>
+            <Grid item xs={3}>
+              <Avatar style={{ height: 50, width: 50 }} src={chefProfilePic} />
+            </Grid>
+            <Grid item container xs={8}>
```
Comment (dproc96) [inline client/src/components/SearchRecipesCard.js]: This is displaying weirdly on my laptop screen, looks fine on my monitor so I think it's the size
Screen Shot 2021-03-31 at 8 14 35 AM

File: client/src/components/SearchRecipesCard.js [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+              <Grid item xs={12}>
+                <Typography
+                  style={{ fontSize: 18, fontWeight: 500 }}
```
Comment (dproc96) [inline client/src/components/SearchRecipesCard.js]: Would be good to pull the inline styles out to the makeStyles function

File: client/src/pages/SearchRecipes.js [hunk 4]
```diff
@@ -0,0 +1,312 @@
...
+        </Grid>
+        <Grid item container spacing={5} xs={9} className={classes.results}>
+          <Grid item xs={12}>
+            <Typography component="h2" variant="h2" style={{ fontSize: 20 }}>
+              {resultsHeader}
+            </Typography>
+          </Grid>
+          {recipes.map((recipe) => {
+            return (
+              <Grid item xs={3}>
```
Comment (dproc96) [inline client/src/pages/SearchRecipes.js]: You should add a unique key here

**Review by dproc96 — COMMENTED** [review]
Had a couple of comments, but this looks like a good set up for the UI

File: client/src/components/SearchRecipesCard.js (hunk 2 above)
Comment (rudream) [reply client/src/components/SearchRecipesCard.js]: I'll make the text smaller

File: client/src/pages/SearchRecipes.js (hunk 4 above)
Comment (rudream) [reply client/src/pages/SearchRecipes.js]: Will do.

File: client/src/components/SearchRecipesCard.js (hunk 3 above)
Comment (rudream) [reply client/src/components/SearchRecipesCard.js]: Can do. I tend to leave styles inline when it's only one or two styles that are only going to be applied to that one text.

File: client/src/components/ChefProfile/ChefSideBar.js [hunk 5]
```diff
@@ -121,19 +121,27 @@ const ChefSideBar = (props) => {
...
+                <Typography variant="h4" style={{ fontWeight: 500 }}>
                   {profile ? profile.name : userInfo.name}
                 </Typography>
               )}
             </Box>
             <Box>
-              <Typography variant="body2">{location}</Typography>
+              <Typography
+                variant="body2"
+                style={{ fontSize: "20px", fontWeight: 600 }}
```
Comment (dproc96) [inline client/src/components/ChefProfile/ChefSideBar.js]: Can you extract these styles as well?

File: client/src/components/SearchRecipesCard.js [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+          <Typography component="h1" variant="h1" className={classes.titleText}>
+            {recipeName}
+          </Typography>
+          <Typography variant="h6" className={classes.price}>
+            ${recipePrice}
+          </Typography>
+        </CardContent>
+        <Divider />
+        <CardContent className={classes.cardContent}>
+          <Grid container>
```
Comment (dproc96) [inline client/src/components/SearchRecipesCard.js]:
```suggestion
          <Grid container spacing={1}>
```
You should be able to use this to add a bit of space between the avatar and name

File: client/src/pages/SearchRecipes.js [hunk 7]
```diff
@@ -0,0 +1,323 @@
...
+
+  const [recipes, setRecipes] = useState([]);
+
+  useEffect(() => {
+    const getInitialRecipes = async () => {
+      try {
+        const res = await axios.get(url);
+        const data = res.data.data;
+        await setRecipes(data);
+        console.log(data);
```
Comment (dproc96) [inline client/src/pages/SearchRecipes.js]: Can you remove this console.log?

Comment (dproc96) [inline client/src/pages/SearchRecipes.js]: Can you remove this console.log?

**Review by dproc96 — APPROVED** [review]
A couple more minor changes, but pending those this is good to merge

**Review by dproc96 — APPROVED** [review]
A couple more minor changes, but pending those this is good to merge

**Pull request merged**